src/main/resources/beam-template.conf

```diff
@@ -268,6 +268,8 @@ beam.agentsim.agents.vehicles.sharedFleets = [
 beam.agentsim.collectEvents=false
 
 beam.agentsim.agents.population.useVehicleSampling = "boolean | false"
+beam.agentsim.agents.population.industryRemovalProbabilty.inputFilePath = ""
+beam.agentsim.agents.population.industryRemovalProbabilty.removeAgent = "boolean | false"
```

Comment:
Please, add some description. I can see that if `removeAgent ` is false, then it will remove all work plans.

Reply:
As per discussion, let's add another parameter to control this functionality: `beam.agentsim.agents.population.industryRemovalProbabilty.enabled  = "boolean | false"`

Reply:
add comment on usability of removeAgent and add control param for removeAgent